src/main/kotlin/no/nav/familie/ef/sak/opplysninger/mapper/BarnMedSamværMapper.kt

```diff
@@ -85,7 +85,8 @@ object BarnMedSamværMapper {
                 harSammeAdresse = matchetBarn.barn?.let {
                     AdresseHjelper.borPåSammeAdresse(it, søkerAdresse)
                 },
-                forelder = pdlAnnenForelder?.let { tilAnnenForelderDto(it, annenForelderFnr) }
+                forelder = pdlAnnenForelder?.let { tilAnnenForelderDto(it, annenForelderFnr) },
+                dødsdato = matchetBarn.barn?.dødsfall?.firstOrNull {it != null}?.dødsdato,
```

Comment:
Trenger man `{it != null}`?
```suggestion
                dødsdato = matchetBarn.barn?.dødsfall?.firstOrNull()?.dødsdato,
```

Reply:
Kan vi bruke `dødsfall?.gjeldende()?.dødsdato` ?

Reply:
(vil ikke at vi bruker `firstOrNull` på pdlListene for å hente ut gjeldende verdi, selv om det gir mening her. Mye lettere å forvalte "samme struktur" senere for å unngå misforståelser :)

Reply:
Fixet i e96c927. Bra dere så det :)